packages/fluentui/docs/src/examples/components/Popup/Visual/PopupExampleContainerTransformed.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { Button, Popup } from '@fluentui/react-northstar';
+import { MoreIcon } from '@fluentui/react-icons-northstar';
+
+const PopupExampleContainerTransformed = () => (
+  <div style={{ transform: 'translate3d(0,0,0)', margin: 200, border: '2px solid red' }}>
+    <Popup content="This popup is rendered next to the trigger." open positionFixed>
```

Comment:
```suggestion
  <div
    style={{
      border: '2px solid red',
      margin: 200,
      height: 400,
      width: 400,
      overflow: 'scroll',
      transform: 'translate3d(0,0,0)',
    }}
  >
    <Popup content="This popup is rendered next to the trigger." inline open positionFixed>
```

We need to update this example as we missed the key thing, `Popup` should be rendered inline as otherwise it will be rendered to `document.body`.